src/modules/orbitdb/inmemory.js

```diff
@@ -6,41 +6,54 @@ const version = 0;
 class ChluInMemoryIndex extends ChluAbstractIndex {
     constructor(){
         const _index = {
-            list: [],
-            data: {}
+            verifiedReviews: {
```

Comment:
Remind me why we can't unify the verified and unverified reviews? Or is that a future work? Given we are doing things without hacking to reach a deadline, should we step back and figure out a way to unify these?

I'd like verified and unverified reviews to be saved in exactly the same manner. In the DR Spec we talk of a flag called "verifiable", if it is false it is an unverified review and we should only check it was a valid verifiable claim signed by Chlu private key. That is our DOS attack defense. If the verifiable flag is true *and* the verifification_method is set to 'chlu' then we should run the chlu protocol verification.

We might need to discuss this a bit, but I think this is an important issue. We should avoid saving the reviews in different format and in different data structures.

Reply:
Yes, this is a very good point.

What we have to figure out before we do can do that:

- a unique identifier for unverified reviews
- storing them in the same format on IPFS, otherwise what happens is that we have the multihash of a review but we don't know how to deserialize it without knowing what format it is in advance. We can do an approach where we try one and then try the other if it fails but it's not very clean

Anyway I think this PR should focus on DIDs and not worry about unverified reviews. I put some preliminary work about unverified reviews, but it's not working yet and I included it just so that I wouldn't have to throw it away